data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/ComplexNetworkAction.java

```diff
@@ -31,16 +30,16 @@
 public class ComplexNetworkAction extends AbstractRemedialAction implements NetworkAction {
 
     @JsonProperty("applicableNetworkActions")
```

Comment:
remove applicable network action and put network action instead

Reply:
ok